content/en/docs/tasks/observability/distributed-tracing/telemetry-api/snips.sh

```diff
@@ -128,10 +127,9 @@ spec:
   meshConfig:
     enableTracing: true
     defaultConfig:
-      tracing: {} # disabled tracing options via `MeshConfig`
+      tracing: {} # disable legacy tracing options via `MeshConfig`
```

Comment:
Do we have a section that speaks on the importance of disabling the legacy tracing option?

Reply:
We can't even prove it's important at this point.  I hope it gets tidied up in the deprecation of it!